packages/contentful--app-scripts/lib/activate/activate-bundle.test.js

```diff
@@ -10,7 +10,8 @@ const mockedSettings = {
 };
 
 describe('activate-bundle', () => {
-  let activateBundle, clientMock, updateStub;
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Why unused?

Reply:
Code was nagging about it, though it's actually used